packages/node_modules/@cerebral/angular/package.json

```diff
@@ -14,6 +14,7 @@
   },
   "homepage": "http://cerebral.github.io/cerebral-website",
   "dependencies": {
+    "@types/mocha": "^2.2.43",
```

Comment:
Are mocha types still necessary after removing the tests?

Reply:
working on adding the test back now